packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1493,4 +1498,25 @@ idx.error.user.not.assigned.to.app = User is not assigned to this application.
 idx.error.code.user_not_assigned = You are not allowed to access this app. To request access, contact an admin.
 
 # Custom Identifiers
-custom.identifiers.userNotResolved = Can't identify your account with {0}. Enter your username.
\ No newline at end of file
+custom.identifiers.userNotResolved = Can't identify your account with {0}. Enter your username.
```

Comment:
For all the argument `{0}`, could you please add a short description. It will give our translator/vendor some more context, so that we can have a better translation quality. Example: line 877